cloudify_marketplace/resources/fix_cloudify_ip_configurations

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
```

Comment:
in the long run, we should probably find a more robust way to configure these. man.. this is ugly :)